tools/sigh.ts

```diff
@@ -82,7 +82,7 @@ const cleanFiles = ['manifest-railroad.html', eslintCache];
 const cleanDirs = ['shell/build', 'shells/lib/build', 'build', 'dist', 'src/gen', 'test-output', 'user-test', coverageDir];
 
 // RE pattern to exclude when finding within project source files.
-const srcExclude = /\b(node_modules|deps|build|gen|dist|third_party|hackathon|javaharness|Kotlin|particles[/\\]Native)\b/;
+const srcExclude = /\b(node_modules|deps|build|gen|dist|third_party|hackathon|server|javaharness|Kotlin|particles[/\\]Native)\b/;
```

Comment:
Ah good catch, wonder how this didn't break before...

Reply:
I think it was just being linted, but the .js import style doesn't work there.. Seemed easier to just exclude it than to continue linting it.